### DISCUSSION TITLE
Preview site is down

### DISCUSSION THREAD

**Discussion by TheSpaceDork** [react-icons/react-icons#971] [General]
Preview site has not opened for weeks if not months and It's really not easy to just guess the icon I'm looking for, I have to keep importing and deleting until I find what I need ....It's really slowing me down . Could something be done to fix it please?

**Comment by nolanleung** [comment]
Which URL are you using? Clicking on the readme header and "View the documentation" take you to  https://react-icons.github.io/react-icons/

**Reply by arianetsuru** [reply]
This url is still not working, the site doesn't load

**Reply by vittxr** [reply]
I got timeout. I think it's related to `.io` domain (https://every.to/p/the-disappearance-of-an-internet-domain)

![image](https://github.com/user-attachments/assets/c26a2395-f376-460c-ab34-a5d8e0595a2b)

**Reply by Henok-Enyew** [reply]
> Which URL are you using? Clicking on the readme header and "View the documentation" take you to https://react-icons.github.io/react-icons/

i am experienciing the same issue this week, i am not able to access the website

**Comment by maotora** [comment]
Goddamit, it's down again.
